### PR TITLE
Integrate github.query with existing configuration model.

### PR DESCRIPTION
**Pull request by ryneeverett**

### CODE REVIEW

File: tests/test_github.py [hunk 1]
```diff
@@ -143,6 +143,8 @@ class TestGithubIssueQuery(AbstractServiceTest, ServiceTest):
         'github.password': 'arbitrary_password',
         'github.username': 'arbitrary_username',
         'github.query': 'is:open reviewer:octocat',
+        'github.include_user_repos': 'False',
+        'github.include_user_issues': 'False',
```
Comment (djmitche) [inline tests/test_github.py:147]: hm, I don't think this is right..

Comment (ryneeverett) [reply tests/test_github.py:147]: Did you see my latest comment in [452](https://github.com/ralphbean/bugwarrior/pull/452)?

> The changes I made to the test configuration explain how to achieve the behaviour you want. I can understand if you wouldn't be thrilled about having to set those flags to False, but to me that's a better trade off than having the query option dynamically change the default values to False, which is what this PR currently effectively does. I might be receptive to changing those default values to False, but dynamic default values seem too confusing.

Comment (djmitche) [reply tests/test_github.py:147]: oh, never mind me

Comment (djmitche) [reply tests/test_github.py:147]: Yeah, I didn't see `include_user_issues` in the source at all, but I fat-fingered the grep..

**Pull request merged**